src/app/notifications/notifications.component.ts

```diff
@@ -8,29 +8,32 @@ declare var $: any;
 export class NotificationsComponent implements OnInit {
 
   constructor() { }
-  showNotification(from, align){
-      const type = ['','info','success','warning','danger'];
+  showNotification(from, align) {
```

Comment:
interesting notification handling

Reply:
I removed the whole component... not used either... some legacy code i think.